src/Pecee/Http/Input/InputHandler.php

```diff
@@ -292,24 +343,16 @@ public function get(string $index, ?string $defaultValue = null)
      */
     public function all(array $filter = []): array
     {
-        $output = $_GET;
+        $output = $this->originalParams + $this->originalPost + $this->originalFile;
```

Comment:
This does not work in my tests:
```
$arr1 = array('a', 'b');
$arr2 = array('c', 'd');

echo json_encode($arr1+$arr2);
```
-> ["a","b"]

Reply:
Language operators are not the same as array_merge. It will prioritize the keys from left to right (in your case they all share same key 0) so the first arrays value will be used.

>The + operator returns the right-hand array appended to the left-hand array; for keys that exist in both arrays, the elements from the left-hand array will be used, and the matching elements from the right-hand array will be ignored.

Try changing your test to arrays with key names.

Reply:
Ah, I got your point right^^ 👍🏻